packages/playwright/src/index.ts

```diff
@@ -97,6 +98,19 @@ function getViewportSize(page: Page) {
   return viewportSize;
 }
 
+// Set viewport size and wait for the visual viewport to match
```

Comment:
JSDoc comment is better, so you can benefit from it in Intellisense autocompletion.